python/taichi/core/__init__.py

```diff
@@ -2,6 +2,7 @@
 from taichi.core.settings import *
 from taichi.core.record import *
 from taichi.core.logging import *
+from taichi.core.primitive_types import *
```

Comment:
Not sure how I feel about this... It's bad to do wildcard import. On the other hand, there are arguments saying that doing this in `__init__.py` is an exception, in order to lift the symbols into the package level. If not written this way, we would have to write `from taichi.x import float32, f32, float64, f64, ..., u64` twice, which is quite repetitive (and equally hard to maintain). What's your opinion :) ?

Reply:
I think the current solution should be good, given `taichi.core.primitive_types` already clearly defined the `__all__` variable, listing the names to export :-)